### PR TITLE
Add mgs fs to stats service

### PR DESCRIPTION
**Pull request by johnsonw**
Add mgs fs to stats service

---
This change is [Reviewable](https://reviewable.io/reviews/whamcloud/integrated-manager-for-lustre/2147)

### CODE REVIEW

**Comment by johnsonw** [comment]
testing with zfs:
```
chroma=> select * from targets;
  state  |     name      | active_host_id | host_ids | filesystems |         uuid         |     mount_path
---------+---------------+----------------+----------+-------------+----------------------+---------------------
 mounted | zfsmo-MDT0001 |              2 | {2,1}    | {zfsmo}     | 13784013625799897582 | /lustre/zfsmo/mdt1
 mounted | zfsmo-OST000a |              4 | {4,3}    | {zfsmo}     | 9913219417477061819  | /lustre/zfsmo/ost10
 mounted | zfsmo-OST000b |              4 | {4,3}    | {zfsmo}     | 2070171407754959751  | /lustre/zfsmo/ost11
 mounted | zfsmo-OST0000 |              3 | {3,4}    | {zfsmo}     | 10129430661856804168 | /lustre/zfsmo/ost0
 mounted | zfsmo-OST0001 |              3 | {3,4}    | {zfsmo}     | 10468302977429261680 | /lustre/zfsmo/ost1
 mounted | zfsmo-OST0002 |              3 | {3,4}    | {zfsmo}     | 1194721771181014559  | /lustre/zfsmo/ost2
 mounted | zfsmo-OST0003 |              3 | {3,4}    | {zfsmo}     | 8185733351275163615  | /lustre/zfsmo/ost3
 mounted | zfsmo-OST0004 |              3 | {3,4}    | {zfsmo}     | 15397621233153631351 | /lustre/zfsmo/ost4
 mounted | zfsmo-OST0005 |              3 | {3,4}    | {zfsmo}     | 13308332230825550895 | /lustre/zfsmo/ost5
 mounted | zfsmo-OST0006 |              3 | {3,4}    | {zfsmo}     | 14931732989337873503 | /lustre/zfsmo/ost6
 mounted | zfsmo-OST0007 |              3 | {3,4}    | {zfsmo}     | 7642174993653384461  | /lustre/zfsmo/ost7
 mounted | zfsmo-OST0008 |              3 | {3,4}    | {zfsmo}     | 992934174923176715   | /lustre/zfsmo/ost8
 mounted | zfsmo-OST0009 |              3 | {3,4}    | {zfsmo}     | 7817761909122678528  | /lustre/zfsmo/ost9
 mounted | MGS           |              1 | {1,2}    | {zfsmo}     | 2077040282944542639  | /lustre/zfsmo/mgs
 mounted | zfsmo-MDT0000 |              1 | {1,2}    | {zfsmo}     | 2148002688826283218  | /lustre/zfsmo/mdt0
 mounted | zfsmo-OST000c |              4 | {4,3}    | {zfsmo}     | 15616620734974383223 | /lustre/zfsmo/ost12
 mounted | zfsmo-OST000d |              4 | {4,3}    | {zfsmo}     | 3944579706361897949  | /lustre/zfsmo/ost13
 mounted | zfsmo-OST000e |              4 | {4,3}    | {zfsmo}     | 16832700998462991308 | /lustre/zfsmo/ost14
 mounted | zfsmo-OST000f |              4 | {4,3}    | {zfsmo}     | 11932332147125695175 | /lustre/zfsmo/ost15
 mounted | zfsmo-OST0010 |              4 | {4,3}    | {zfsmo}     | 11027676443614933072 | /lustre/zfsmo/ost16
 mounted | zfsmo-OST0011 |              4 | {4,3}    | {zfsmo}     | 5415649341819134455  | /lustre/zfsmo/ost17
 mounted | zfsmo-OST0012 |              4 | {4,3}    | {zfsmo}     | 15784332873808445832 | /lustre/zfsmo/ost18
 mounted | zfsmo-OST0013 |              4 | {4,3}    | {zfsmo}     | 14584183703682808910 | /lustre/zfsmo/ost19
```

**Review by jgrund — DISMISSED** [review]

File: docker/wait-for-dependencies-postgres.sh [hunk 1]
```diff
@@ -10,6 +10,7 @@ done
 TMP=$PROXY_HOST
 TMP2=$AMQP_BROKER_HOST
 TMP3=$DB_HOST
+TMP4=$INFLUXDB_HOST
```
Comment (jgrund) [inline docker/wait-for-dependencies-postgres.sh:13]: Why is this needed?

File: docker/wait-for-dependencies.sh [hunk 2]
```diff
@@ -10,6 +10,7 @@ done
 TMP=$PROXY_HOST
 TMP2=$AMQP_BROKER_HOST
 TMP3=$DB_HOST
+TMP4=$INFLUXDB_HOST
```
Comment (jgrund) [inline docker/wait-for-dependencies.sh:13]: Why is this needed?

File: docker/wait-for-dependencies-postgres.sh (hunk 1 above)
Comment (johnsonw) [reply docker/wait-for-dependencies-postgres.sh:13]: I'll try to keep this short but it's a rather involved answer. In my initial push the rpm build worked fine because the influxdb client was able to connect to the host over 127.0.0.1. The docker build, it turns out, was also using 127.0.0.1. To resolve this, I added a variable to hold the value of the INFLUXDB_HOST on both the iml-stats and iml-device services. This was added to the docker-compose file as an environment variable and it fixed the issue for docker. But then I noticed the RPM build was broken and it turns out it broke because the INFLUXDB_HOST variable needed to be included in the print-settings output in order for the rpm service to load the environment variable. So I added it to the print-settings list and sure enough that fixed the rpm build. But now, the docker build was broke again... While inspecting the docker containers I saw that the INFLUXDB_HOST variable was set as expected but looking at /proc/1/environ it was apparent that the INFLUXDB_HOST was 127.0.0.1. I then realized that the environment variables in the settings file will overwrite the environment variables specified in the docker-compose file. To resolve this, I needed to update`wait-for-dependencies.sh` because it was loading the settings file and then overwriting specific environment variables for docker, such as PROXY_HOST and DB_HOST. Now, INFLUXDB_HOST is overwritten in the wait-for-settings file AFTER the settings file is loaded. If this is not done, the docker container will not retain the value that is set in the docker-compose environment block.

File: docker/wait-for-dependencies.sh (hunk 2 above)
Comment (johnsonw) [reply docker/wait-for-dependencies.sh:13]: Same as the above answer.

File: docker/wait-for-dependencies-postgres.sh (hunk 1 above)
Comment (johnsonw) [reply docker/wait-for-dependencies-postgres.sh]: We've discussed a new solution. Resolving this conversation for now.

**Review by ip1981 — APPROVED** [review]
